helpers.go

```diff
@@ -26,6 +26,16 @@ type Optional interface {
 	IsDefined() bool
 }
 
+// UnknownsUnmarshaler provides a method to unmarshal unknown struct fileds and save them as you want
+type UnknownsUnmarshaler interface {
+	UnmarshalUnknown(key string, value interface{})
```

Comment:
It's better to allow returning an error, it may be useful for custom validation/unmarshalling.

Reply:
in this PR https://github.com/mailru/easyjson/pull/267/files i changed this interface to 
```go
type UnknownsUnmarshaler interface {
	UnmarshalUnknown(in *jlexer.Lexer, key string)
}
```

so developers can use jlexer.Lexer to put errors